homework/index.js

```diff
@@ -36,10 +19,10 @@ function createAndAppend(name, parent, options = {}) {
     const value = options[key];
```

Comment:
A bit confused on this file... are there two index.js files in the pull request?